pkg/collector/corechecks/snmp/checkconfig/config_metric.go

```diff
@@ -81,64 +85,6 @@ type MetricsConfig struct {
 	Options    MetricsConfigOption `yaml:"options"`
 }
 
-// GetTags retrieve tags using the metric config and values
-func (mtcl MetricTagConfigList) GetTags(fullIndex string, values *valuestore.ResultValueStore) []string {
```

Comment:
note for reviewers: `GetTags` has been moved to `pkg/collector/corechecks/snmp/report/report_utils.go` 

only this section changed:

https://github.com/DataDog/datadog-agent/pull/10155/files#diff-beafef85ae39c3cfe92bed173e53fae25163058b8c86a08a0cb9405b6681c04eR98-R100

```
			newIndex := fullIndex
			newIndex = newIndexUsingIndexFromOidValue(newIndex, metricTag.Column.IndexFromOidValue.OID, values)
			newIndex = newIndexUsingTransformIndex(newIndex, metricTag.IndexTransform)
```